tests/benchmark/config.json

```diff
@@ -37,32 +36,9 @@
       "size": "9.9M"
     },
     {
-      "conn_id": "bigquery",
-      "file_type": "ndjson",
-      "name": "one_gb",
-      "path": "gs://astro-sdk/benchmark/trimmed/stackoverflow/stackoverflow_posts_1g.ndjson",
-      "rows": 385817,
-      "size": "1G"
```

Comment:
Why remove this from here?

Reply:
@kaxil I'm not sure hwo that happened. I added it back in.